### PR TITLE
add announcements about pmdk and miniasync releases

### PR DESCRIPTION
**Pull request by lplewa**
https://lplewa.github.io//announcements/2022/pmdk-v1-12-0-release/
https://lplewa.github.io//announcements/2022/miniasync-v0-1-0-release/

This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)

### CODE REVIEW

**Comment by pbalcer** [comment]
can you link a preview?

File: content/announcements/MiniAsync-v0-1-0-release.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# Featured. Specify true or false to show on homepage
+featured: 
+---
+
+We've just released a first experimental version of miniasyc library. I'd like to thank everyone who contributed.
+
+The library provides the API for asynchronous memory operations
+through the use of features and runtime mechanisms and independence
+in terms of hardware by implementing a virtual data mover.
```
Comment (pbalcer) [inline content/announcements/MiniAsync-v0-1-0-release.md:34]: `The library provides a framework for the composition and execution of asynchronous tasks in C, as well as an abstract interface for hardware-accelerated data movement.`

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)* status: all files reviewed, 5 unresolved discussions (waiting on @lplewa)

___
*[`content/announcements/MiniAsync-v0-1-0-release.md` line 18 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4QiTLU7vzQHaTURMrx:-N4QiTLU7vzQHaTURMry:b-ok62mm) ([raw file](https://github.com/pmem/pmem.github.io/blob/4d50bdfcb9a4dfefb3e3ed2db4485c8a1991f6e7/content/announcements/MiniAsync-v0-1-0-release.md#L18)):*
> ```Markdown
> 
> # Event image
> image: "https://opengraph.githubassets.com/706d2635f355e31fa44b50d0e337af4ce73086e7/pmem/pmdk"
> ```

`pmem/pmdk`?
___
*[`content/announcements/MiniAsync-v0-1-0-release.md` line 44 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4QiIL_Cohi8UkwRKkC:-N4QiIL_Cohi8UkwRKkD:b8s4k9b) ([raw file](https://github.com/pmem/pmem.github.io/blob/4d50bdfcb9a4dfefb3e3ed2db4485c8a1991f6e7/content/announcements/MiniAsync-v0-1-0-release.md#L44)):*
> ```Markdown
> This relase is available on github https://github.com/pmem/miniasync/releases/tag/0.1.0
> 
> Learn more about miniasync library from our blogs:
> ```

you can add hashmap example blog post (https://pmem.io/blog/2022/06/basic-asynchronous-hashmap-with-miniasync-library/) or just link to our blogs' tag: https://pmem.io/tags/miniasync/
___
*[`content/announcements/MiniAsync-v0-1-0-release.md` line 45 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4Qhar5B7irSOxZu2wD:-N4Qhar5B7irSOxZu2wE:b4t5460) ([raw file](https://github.com/pmem/pmem.github.io/blob/4d50bdfcb9a4dfefb3e3ed2db4485c8a1991f6e7/content/announcements/MiniAsync-v0-1-0-release.md#L45)):*
> ```Markdown
> 
> Learn more about miniasync library from our blogs:
> [Leveraging asynchronous hardware accelerators for fun and profit](https://pmem.io/blog/2022/02/leveraging-asynchronous-hardware-accelerators-for-fun-and-profit/)
> ```

please drop here and below `https://pmem.io`
___
*[`content/announcements/PMDK-v1-12-0-release.md` line 36 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4Qir3o7kr53_yIN5nU:-N4Qir3o7kr53_yIN5nV:b-7fhfkx) ([raw file](https://github.com/pmem/pmem.github.io/blob/4d50bdfcb9a4dfefb3e3ed2db4485c8a1991f6e7/content/announcements/PMDK-v1-12-0-release.md#L36)):*
> ```Markdown
> and is no longer recommended. Those interested in a remote
> persistent memory support should use the new
> rpma (https://github.com/pmem/rpma) library.
> ```

I believe you can link here to pmem.io (https://pmem.io/rpma/) - `[...](/rpma/]` to be precise.

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)* status: all files reviewed, 8 unresolved discussions (waiting on @lplewa and @pbalcer)

___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 45 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4k_JVT86v4tGGdmMNP:-N4k_JVT86v4tGGdmMNQ:b-oydlz1) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L45)):*
> ```Markdown
> 
> In this release, we also introduce asynchronous data movement operations in
> the libpmem2 library. This API is marked as experimental, and to enable this, you
> ```

enable it
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 47 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kaXYX4Z0lo9YLiBHS:-N4kaXYX4Z0lo9YLiBHT:bojndhl) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L47)):*
> ```Markdown
> the libpmem2 library. This API is marked as experimental, and to enable this, you
> have to `#define PMEM2_USE_MINIASYNC` before `#include <libpmem2.h>`.
> In doing so, you also have to install the miniasync library, which this API requires.
> ```

By doing so
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 47 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kbbjl4JOJtwlSIHqI:-N4kbbjl4JOJtwlSIHqJ:bqmkfr7) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L47)):*
> ```Markdown
> the libpmem2 library. This API is marked as experimental, and to enable this, you
> have to `#define PMEM2_USE_MINIASYNC` before `#include <libpmem2.h>`.
> In doing so, you also have to install the miniasync library, which this API requires.
> ```

\+ link to miniasync repo or readme
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 55 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kaDaICO-woHR8wM2h:-N4kaDaICO-woHR8wM2i:b-8rdpt6) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L55)):*
> ```Markdown
> memcpy variant that uses this instruction. By default, PMDK will use this instruction if
> it is available. If not, it will try to use AVX512f, then AVX, and if none of them is available
> PMDK will fall back to the standard SSE2 instructions. PMDK performs this decision transparently, and we are not guaranteed 64b atomicity in the PMDK memory manipulation functions, even if
> ```

`we are not guaranteeing`
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 55 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kaMH714D8nNbXXSFM:-N4kaMH714D8nNbXXSFN:b4pwzh2) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L55)):*
> ```Markdown
> memcpy variant that uses this instruction. By default, PMDK will use this instruction if
> it is available. If not, it will try to use AVX512f, then AVX, and if none of them is available
> PMDK will fall back to the standard SSE2 instructions. PMDK performs this decision transparently, and we are not guaranteed 64b atomicity in the PMDK memory manipulation functions, even if
> ```

performs decision? perhaps `makes decision`
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 56 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kajar3EH8e22hy5Am:-N4kajar3EH8e22hy5An:b2q3331) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L56)):*
> ```Markdown
> it is available. If not, it will try to use AVX512f, then AVX, and if none of them is available
> PMDK will fall back to the standard SSE2 instructions. PMDK performs this decision transparently, and we are not guaranteed 64b atomicity in the PMDK memory manipulation functions, even if
> MOVEDIR64B is available in the system. The team will continue to work to utilize this
> ```

to fully utilize
(drop "fully" at the end of this sentence, I guess)
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 61 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kb0_d92H4Gnko0OV_:-N4kb0_d92H4Gnko0OVa:bdo6nan) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L61)):*
> ```Markdown
> Other changes and notable bug fixes:
> - Adds new API to machine safe read/write operations in the pmem2 library.
> - Adds experimental support for RISC-V.
> ```

here you have regular sentences and below commits with tags/areas (pls unify - fix one or another)

**Review by pbalcer — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)* status: all files reviewed, 9 unresolved discussions (waiting on @lplewa and @lukaszstolarczuk)

___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 45 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N50F7w300lIAnQ7vcQx:-N50F7w300lIAnQ7vcQy:b-tb9ez7) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L45)):*
> ```Markdown
> 
> In this release, we also introduce asynchronous data movement operations in
> the libpmem2 library. This API is marked as experimental, and to enable this, you
> ```

in libpmem2 or pmem2 library
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 47 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kaXYX4Z0lo9YLiBHS:-N50FFi88fvm4JoV-wuQ:b-cvw7c8) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L47)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

By doing so
</blockquote></details>

I'm not sure. Maybe "Additionally, this API requires that you also have the miniasync library installed."
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 53 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N50FcVM8ATqG8lsFq98:-N50FcVM8ATqG8lsFq99:b-sbmgxg) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L53)):*
> ```Markdown
> This release also introduces support for movdir64b instruction for memory operation
> functions in the libpmem2 library. movdir64 is a new CPU instruction that allows writing 64 bytes directly to the memory atomically. In PMDK 1.12, we implemented an additional
> memcpy variant that uses this instruction. By default, PMDK will use this instruction if
> ```

movdir64b is really default? @kswiecicki can you chime in?

**Comment by kswiecicki** [comment]
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 53 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N50FcVM8ATqG8lsFq98:-N59IFIv0nc-i34syRlA:b-alwh4) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L53)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

movdir64b is really default? @kswiecicki can you chime in?
</blockquote></details>

movdir64b is turned off currently. To turn it on by default we need to remove this line from `x86_64/sources.inc` file:
`MOVDIR64B_AVAILABLE=n # XXX delete this when movdir64b is ready to be upstreamed`

**Review by pbalcer — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)* status: all files reviewed, 9 unresolved discussions (waiting on @kswiecicki, @lplewa, and @lukaszstolarczuk)

___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 53 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N50FcVM8ATqG8lsFq98:-N5AM9RHES4cmJZGIdWB:b-bx8z81) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L53)):*
<details><summary><i>Previously, kswiecicki (Krzysztof Święcicki) wrote…</i></summary><blockquote>

movdir64b is turned off currently. To turn it on by default we need to remove this line from `x86_64/sources.inc` file:
`MOVDIR64B_AVAILABLE=n # XXX delete this when movdir64b is ready to be upstreamed`
</blockquote></details>

Hm, I was hoping it was a runtime switch (e.g., environment variable), instead of a compile-time one.

Anyway, for now, I'd just remove this whole section from this article and make it either default or a runtime option in the next release. There's no point in talking about a feature no one can't use.

**Review by lplewa — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)* status: 1 of 2 files reviewed, 9 unresolved discussions (waiting on @lukaszstolarczuk and @pbalcer)

___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 45 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4k_JVT86v4tGGdmMNP:-N5AK61-5CnlPKCcwF2L:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L45)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

enable it
</blockquote></details>

Done.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 45 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N50F7w300lIAnQ7vcQx:-N5AK9pX8QRWlLkGFsll:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L45)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

in libpmem2 or pmem2 library
</blockquote></details>

Done.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 47 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kaXYX4Z0lo9YLiBHS:-N5ALyaQ9Xpz9b90rz59:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L47)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

I'm not sure. Maybe "Additionally, this API requires that you also have the miniasync library installed."
</blockquote></details>

Done.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 47 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kbbjl4JOJtwlSIHqI:-N5AM6_aEweTMCw217Yv:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L47)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

\+ link to miniasync repo or readme
</blockquote></details>

Done.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 53 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N50FcVM8ATqG8lsFq98:-N5AMIRJ29SOlwjDBYjF:b-uzec3w) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L53)):*
<details><summary><i>Previously, kswiecicki (Krzysztof Święcicki) wrote…</i></summary><blockquote>

movdir64b is turned off currently. To turn it on by default we need to remove this line from `x86_64/sources.inc` file:
`MOVDIR64B_AVAILABLE=n # XXX delete this when movdir64b is ready to be upstreamed`
</blockquote></details>

well, this means that we dont have movdir64b support?
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 55 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kaDaICO-woHR8wM2h:-N5AMEN3DFuvfyGBx6TD:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L55)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

`we are not guaranteeing`
</blockquote></details>

Done.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 55 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kaMH714D8nNbXXSFM:-N5AMSyZ8e5qTypGjSAB:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L55)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

performs decision? perhaps `makes decision`
</blockquote></details>

Done.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 56 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kajar3EH8e22hy5Am:-N5AMYIt-5ofuXdqHbgj:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L56)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

to fully utilize
(drop "fully" at the end of this sentence, I guess)
</blockquote></details>

Done.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 61 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N4kb0_d92H4Gnko0OV_:-N5ANarb5cq1WEWbZnCb:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L61)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

here you have regular sentences and below commits with tags/areas (pls unify - fix one or another)
</blockquote></details>

Done.

**Review by lplewa — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)* status: 1 of 2 files reviewed, 9 unresolved discussions (waiting on @lukaszstolarczuk and @pbalcer)

___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 53 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N50FcVM8ATqG8lsFq98:-N5AO2kq9-fbuMRB-nSP:b-7n1l8) ([raw file](https://github.com/pmem/pmem.github.io/blob/e11f023e7767cb06ca3ee4fd02a50ea244874467/content/announcements/2022/PMDK-v1-12-0-release.md#L53)):*
<details><summary><i>Previously, lplewa (Łukasz Plewa) wrote…</i></summary><blockquote>

well, this means that we dont have movdir64b support?
</blockquote></details>

@pbalcer I agree- I removed this section.

**Review by pbalcer — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249#-:-N5APE1n5BBfcLzBjBcA:bnfp4nl)* status: all files reviewed (commit messages unreviewed), 9 unresolved discussions (waiting on @lplewa and @lukaszstolarczuk)

___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 6 at r4](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N5AOz4u9UyYx0RAScoO:-N5AOz4u9UyYx0RAScoP:bj16ds8) ([raw file](https://github.com/pmem/pmem.github.io/blob/705c5e6e085452cd863091d3129dedfda6244ce5/content/announcements/2022/PMDK-v1-12-0-release.md#L6)):*
> ```Markdown
> 
> # Creation date
> date: 2022-05-25T16:05:26+02:00
> ```

update to today's date (00:00 clock time)
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 49 at r4](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N5APA9tAqd6T4FAJHC4:-N5APA9tAqd6T4FAJHC5:bltiu2v) ([raw file](https://github.com/pmem/pmem.github.io/blob/705c5e6e085452cd863091d3129dedfda6244ce5/content/announcements/2022/PMDK-v1-12-0-release.md#L49)):*
> ```Markdown
> Additionally, this API requires that you also have the [miniasync library](https://github.com/pmem/miniasync) installed.
> To learn more about this feature, miniasync library, and our overall approach to the
> asynchronous programming with persistent memory, please read out [blog posts](/tags/miniasync/)
> ```

read our

**Review by lplewa — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/249)* status: 1 of 2 files reviewed, 9 unresolved discussions (waiting on @lukaszstolarczuk and @pbalcer)

___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 6 at r4](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N5AOz4u9UyYx0RAScoO:-N5AWpLd8TDNFMv9mnTt:ba72bvt) ([raw file](https://github.com/pmem/pmem.github.io/blob/705c5e6e085452cd863091d3129dedfda6244ce5/content/announcements/2022/PMDK-v1-12-0-release.md#L6)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

update to today's date (00:00 clock time)
</blockquote></details>

this is the release date, and it will look better on the webpage with such a date.
___
*[`content/announcements/2022/PMDK-v1-12-0-release.md` line 49 at r4](https://reviewable.io/reviews/pmem/pmem.github.io/249#-N5APA9tAqd6T4FAJHC4:-N5AWQ8W7VzBe6EoPr5Q:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/705c5e6e085452cd863091d3129dedfda6244ce5/content/announcements/2022/PMDK-v1-12-0-release.md#L49)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

read our
</blockquote></details>

Done.

**Pull request merged**